subworkflows/local/genotype.nf

```diff
@@ -368,7 +400,8 @@ workflow GENOTYPE {
                 .map {
                     combo_meta, metas, bams, bais ->
                     def new_map = [:]
-                    def ids = metas.collect { meta -> meta.sample_id }
+                    // ids will either be sampleID or libraryID depending on desired input for genotyping (default is use meta.sample_id, for merged sample data)
+                    def ids = params.genotyping_use_unmerged_libraries ? metas.collect { meta -> meta.library_id } : metas.collect { meta -> meta.sample_id }
```

Comment:
Doesn't line 88 already take care of this?

Reply:
88 is only executed for pileupcaller genotyping